GNUmakefile

```diff
@@ -464,7 +507,7 @@ clean:
 #######################################################################
 
 define make-lib
-    $(AR) rcs $@ $(subst $(LIBNAME_GTKD),,$^)
+    $(AR) rcs $@ $(subst $(LIBNAME_GTKD4),,$^)
```

Comment:
Based on how you are calling `make-lib` on line 208 this change isn't needed, and it breaks building the other libraries.